app/javascript/viewer/uv_manager.js

```diff
@@ -137,6 +138,26 @@ export default class UVManager {
     }
   }
 
+  bindUvClick () {
+    const uvElement = document.getElementById('uv')
+    const uvClick = new Event("uv-click", { bubbles: true });
+
+    uvElement.addEventListener(
```

Comment:
What are the chances we can get a test? I think I wrote one for plausible stuff in the viewer test before that you could use as a model.

Reply:
Done!